virtcontainers/network.go

```diff
@@ -63,6 +64,20 @@ func (n NetInterworkingModel) IsValid() bool {
 	return 0 <= int(n) && int(n) < int(NetXConnectInvalidModel)
 }
 
+// haveVhostNetKernelModule returns true if the vhost_net module exists
+// (either loaded or available to be loaded)
+func haveVhostNetKernelModule() bool {
+	// First, check to see if the vhost module is already loaded
+	if _, err := os.Stat("/sys/module/vhost_net"); !os.IsNotExist(err) {
```

Comment:
This isn't right, I think if Stat() returns error, no matter what the error is, it should return false